afl/Makefile

```diff
@@ -0,0 +1,15 @@
+#	$NetBSD$
+
+KMOD=	kcov_afl
+SRCS=	kcov_afl.c
+
+HAVE_DEBUGCON_PRINTF=0
+
+.if ${HAVE_DEBUGCON_PRINTF} == 1
+CPPFLAGS+=	-I/home/workspace/debugcon_printf/
```

Comment:
We need to replace it with something like:

```
CPPFLAGS_DBGCON!=pkg-config --cflags debugcon_printf
CPPFLAGS+=${CPPFLAGS_DBGCON}
```

Reply:
And we should add .pc file in the debugcon_printf for the next upcoming release... would you like to do it?

Reply:
Is done